Dockerfile.testing

```diff
@@ -42,113 +42,117 @@ RUN mkdir /src
 
 ##------------------------------------------------------------------------------
 FROM builder-base AS builder-py36
+ARG PYVER=36
+COPY --from=py36 /tmp/py${PYVER}-latest /tmp/
 
-RUN mkdir /opt/py36 && chown -R 3036 /opt/py36 /src
-USER 3036
+RUN mkdir /opt/py${PYVER} && chown -R 30${PYVER} /opt/py${PYVER} /src
+USER 30${PYVER}
```

Comment:
This is… a thing 😆

Reply:
Yeah, seems extreme hey?
It's best not to compile things with the root user however.